release-1.6/release-notes-draft.md

```diff
@@ -53,8 +53,49 @@
 ### New Features
 - Add new command "kubectl set selector" ([#38966](https://github.com/kubernetes/kubernetes/pull/38966), [@kargakis](https://github.com/kargakis))
 - Add kubectl create poddisruptionbudget command ([#36646](https://github.com/kubernetes/kubernetes/pull/36646), [@kargakis](https://github.com/kargakis))
+- Add new DaemonSet status fields to kubectl printer and describer.  ([#42843](https://github.com/kubernetes/kubernetes/pull/42843), [@janetkuo](https://github.com/janetkuo))
```

Comment:
@calebamiles it makes sense regarding the "features" meaning and process handling. If there is a feature for 1.6, it has to be reflected in the features repo under 1.6 milestone; otherwise, we have to define it as a "notable" or a "breaking" change.
It's not a problem for me to make edits afterward, but it would be great if people will be aware of it.